drivers/wifi/nrf700x/src/wpa_supp_if.c

```diff
@@ -1819,6 +1819,32 @@ static int nrf_wifi_iftype_change(struct nrf_wifi_vif_ctx_zep *vif_ctx_zep, int
 	return ret;
 }
 
+static int nrf_wifi_wait_for_carrier_status(struct nrf_wifi_vif_ctx_zep *vif_ctx_zep,
```

Comment:
I see that fmac_main.c AP related parts are guarded with `CONFIG_NRF700X_AP_MODE`. Shouldn't we do the same with the wpa_supp bits as well?

Reply:
It is guarded, see line#1728.